Rework of LB.

Nanu308 opened the pull request:
:cl: Nanu308
maptweak: REWORK OF LIGHT BRIGADE.
maptweak: Moved Nuke, AI, Navcom.
maptweak: Made a new way from Briefing to ForePortside Hangar.
maptweak: Added some of that Flare
maptweak: Added better new Engines that do not need cooling
maptweak: Added an Atmos system that functions.
maptweak: Changed Medbay, All should be there. If you miss anything say so.
maptweak: Removed old atmos system
/:cl:

Nanu308 committed:
Rework of LB

Added Atmos, Reworked internal layout and fixes overall to the ship.

Nanu308 committed:
Rework of LB

Rework of LB, Better Atmos system, power, general QOL Changes, Moved ODST prep and Brig, medbay changes, Hangar and fore of ship changes. Etc. Go have a look

Nanu308 committed:
Nuke Moved to Bridge - Navcom+Radio moved to Fore

Nuke Moved to Bridge - Navcom+Radio moved to Fore of the Ship

BDpuffy420 commented:
Travis mad

Nanu308 committed:
Bug Fixes.

General Bug Fixing regarding areas and wires etc.

Nanu308 committed:
More BugFixes, God i hate this bot

Area fixes and potentially camera fix

Nanu308 commented:
### **I WILL KILL THIS BOT**

Nanu308 committed:
More Bug Fixes? 

Hmmm Hmmmmmmmmm Hmmmmmmmmmmmmmmm Sanity Draining

Nanu308 committed:
Merge pull request #1 from HaloSpaceStation/alpha

Udpating Repo

BDpuffy420 commented:
PR still failing due to other reason not related to deso's GM

caelaislinn commented on maps/UNSC_Halberd_Class/areas.dm:
```diff
@@ -1,68 +1,125 @@
-
-/area/destroyer/unsclightbrigade/centralhall
-	name = "UNSC Light Brigade Central Hallway"
+	/area/destroyer/unsclightbrigade
```
This is failing travis, there shouldn't be an indent here

Nanu308 replied:
I believe this have been fixed now

caelaislinn reviewed (changes requested):
Fix that single indent (remove it so it aligns with the edge of the file) and it will fix all the travis failures

Nanu308 committed:
Bugs bugs bugs

*pain *scream *medic

Nanu308 committed:
Merge pull request #2 from HaloSpaceStation/alpha

Merge to New Code

Nanu308 committed:
Quick Hotfix

Quick Hotfix

Nanu308 committed:
Various Minor Fixes

Various Minor Fixes

Nanu308 committed:
Camera Fix

Camera Fix, Pleze work,

Nanu308 committed:
Removed all Invis Cams and placed new ones

Removed all Invis Cams and placed new ones again.

Nanu308 committed:
Hotfix for Issue

Hotfix for Issue with cams and naming in areas.

X0-11 commented:
set to DNM, awaiting further update to changelog

Nanu308 committed:
More Changes /Fixes

Nanu308 committed:
Fixed some AI Vision Issues

Added some stuff for AI's to be able to see shit, and potentially fixed wiring in Engine rooms

Nanu308 committed:
Hotfix

Hotfix on Wires

Nanu308 commented:
I will make a full changelogg tomorrow/today depending if all issues get removed

caelaislinn commented:
Travis failure is achlys map
